package.json

```diff
@@ -40,7 +40,9 @@
     "react-intl": "^2.1.2",
     "react-intl-webpack-plugin": "^0.0.3",
     "rimraf": "^2.4.3",
-    "webpack": "^4.0.0"
+    "webpack": "^4.0.0",
+    "webpack2": "npm:webpack@^2.0.0",
```

Comment:
Do you know if this syntax is supported by `npm`? I'm not sure how I feel about relying on yarn-specific features.

Reply:
Unfortunately it's not supported by `npm`, even though there's been a lot of discussion about it (npm/npm#5499). Alternate version I know of would need `lerna`.